### DISCUSSION TITLE
Nuget csproj support

### DISCUSSION THREAD

**Discussion by Sjord** [aquasecurity/trivy#4220] [Ideas]
C# project files, with the extension .csproj, define dependencies like this:

```
<PackageReference Include="Foo.Bar" Version="1.2.3.4" />
```

I would like for Trivy to scan these dependencies for known vulnerabilities.

See also [NuGet Support · Issue #681 · aquasecurity/trivy](https://github.com/aquasecurity/trivy/issues/681).

**Comment by hazcod** [comment]
@Sjord Sorry, were you ever to merge this into trivy?

**Comment by C1ph3R-s** [comment]
any update in this topic? I would like to use this functionality if it exists.
Thanks

**Reply by DmitriyLewen** [reply]
https://github.com/aquasecurity/go-dep-parser/pull/121 contains information about nuances of `csproj` files support.

**Comment by joelsand** [comment]
I'd also love this functionality
